usr/share/rear/conf/default.conf

```diff
@@ -799,6 +799,13 @@ TIMESYNC_SOURCE=
 # You could redefine it as, e.g. LANG_RECOVER=de_DE@euro
 LANG_RECOVER=C
 
+# LUKS_CRYPTSETUP_OPTIONS contains additional options to cryptsetup which complement auto-detected options.
+# The default setting increases security beyond the level attained by compiled-in cryptsetup defaults.
+# On an embedded system, using the /dev/random random generator may result in (possibly long) delays. In this case, you
+# may set LUKS_CRYPTSETUP_OPTIONS="--iter-time 2000 --use-urandom" instead, but be aware that this produces a
+# low-quality master encryption key. For details, see the cryptsetup(8) manual page.
+LUKS_CRYPTSETUP_OPTIONS="--iter-time 2000 --use-random"
+
```

Comment:
Restoring virtual machines lacks entropy, so it might happen that during restore of VM ReaR will just "sit and wait" ..

Reply:
True. Secure by default, but may not cover every use case. User should re-configure `LUKS_CRYPTSETUP_OPTIONS` in these cases. Whether it's a good idea to use encryption inside entropy-lacking environments is another question.